lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,50 @@
+module Liftoff
+  class SchemeBuilder
+
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_scheme # generates the default scheme
+    
+      # generates custom schemes
```

Comment:
Thoughts on removing the comment? We generally prefer to not have them and let the code speak for itself. I think the code here is readable enough that the comment isn't necessary. :)